configs/tiny/OLMo-150M.yaml

```diff
@@ -78,9 +74,10 @@ save_num_unsharded_checkpoints_to_keep: -1
 
 load_path: null
 
-max_duration: 2ep
+max_duration: 1ep
+stop_at: 406_934
```

Comment:
Do you need both `max_duration` and `stop_at`?

Reply:
yes, from what I have observed and Dave mentioned the training goes past max_duration if stop_at is not set